test/test_helper.rb

```diff
@@ -25,6 +25,67 @@
 
 require 'sucker_punch/testing/inline'
 
+# Mock up vault client
+class VaultClient
+  def logical
+    @logical ||= Logical.new
+  end
+
+  def self.vault_response_object(data)
+    Response.new(data)
+  end
+
+  class Response
+    attr_accessor :lease_id, :lease_duration, :renewable, :data, :auth
+    def initialize(data)
+      self.lease_id = nil
+      self.lease_duration = nil
+      self.renewable = nil
+      self.auth = nil
+      self.data = data
+    end
+
+    def to_h
+      instance_values.symbolize_keys
+    end
+  end
+
+  class Logical
```

Comment:
works for now, I'll re-write this stuff.
https://zendesk.atlassian.net/browse/PAAS-151